WebSocketTest/Programs/TcpServer.cs

```diff
@@ -3,16 +3,16 @@
 using System.Net;
 using System.Threading;
 using WebSocketTest.ConnectionHandlers;
-using WebSocketTest.Datatypes;
+using WebSocketTest.Models.Clients;
 
 namespace WebSocketTest.Programs
 {
-	static class TcpServer
+    internal static class TcpServer
 	{
-		static private readonly IPEndPoint endpoint = new IPEndPoint(IPAddress.Any, 8080);
+		private static readonly IPEndPoint Endpoint = new IPEndPoint(IPAddress.Any, 8079); // TODO: auto free port finder 
```

Comment:
Write an issue instead of writing a comment in code so it is documented as a possible improvement. (revert port change / comment)